frontend/src/app/page.tsx

```
@@ -1,194 +1,186 @@
-"use client"
-import type React from "react"
 import Link from "next/link"
 import Image from "next/image"
-import { ArrowRight, MapPin, Wind, BarChart3, BrainCircuit, Shield } from "lucide-react"
-import Navigation from "@/components/navigation/navigation"
+import { Button } from "@/components/ui/button"
+import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle } from "@/components/ui/card"
+import { MapPin, Map, BarChart2, FileText, Info } from "lucide-react"
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)